### PR TITLE
Varvara/feature/addresses

### PR DESCRIPTION
**Pull request by varvarashcherbyna7**

### CODE REVIEW

File: app/schema/location.py [hunk 1]
```diff
@@ -1,6 +1,14 @@
+import enum
 from pydantic import BaseModel, ConfigDict
 
 
+class Type(enum.Enum):
+    REGION_CENTER = "region_center"
+    RAYON_CENTER = "rayon_center"
+    CITY = "city"
+    VILLAGE = "village"
```
Comment (marynag) [inline app/schema/location.py:9]: Is there a type for urban-type village (смт) settlements?

File: data/locations.json [hunk 2]
```diff
@@ -0,0 +1,108 @@
+{
+  "locations": [
+    {
+      "id": 1,
+      "uuid": "0e5c7eba-13fd-42b8-a4ae-5d973958a160"
```
Comment (Taraslysak) [inline data/locations.json:5]: Why do we need hard-coded uuids?

File: test_api/test_locations.py [hunk 3]
```diff
@@ -23,3 +23,15 @@ def test_get_locations(client: TestClient, full_db: Session):
...
     assert len(db_regions) == len(locations)
+
+
+@pytest.mark.skipif(not CFG.IS_API, reason="API is not enabled")
+def test_get_address(
+    client: TestClient,
+    db: Session,
+    auth_header: dict[str, str],
+):
+    response = client.get(
```
Comment (Taraslysak) [inline test_api/test_locations.py:35]: This test does not test any business logic. Please improve it

**Review by Taraslysak — CHANGES_REQUESTED** [review]

File: data/locations.json (hunk 2 above)
Comment (varvarashcherbyna7) [reply data/locations.json:5]: I took part of the real data that we filled in, and this link with the locations, I need it to fill in the city and address models

**Pull request merged**